root/static/scripts/edit/confirmNavigationFallback.js

```diff
@@ -9,43 +9,43 @@
 import MB from '../common/MB';
 
 MB.confirmNavigationFallback = function () {
-    /*
-     * Every major browser supports onbeforeunload expect Opera. (This says
-     * Opera 12 supports it, but it doesn't, at least not <= 12.10.)
-     * https://developer.mozilla.org/en-US/docs/DOM/window.onbeforeunload
-     */
-    if (window.onbeforeunload !== undefined) {
-        return;
-    }
+  /*
+   * Every major browser supports onbeforeunload expect Opera. (This says
+   * Opera 12 supports it, but it doesn't, at least not <= 12.10.)
+   * https://developer.mozilla.org/en-US/docs/DOM/window.onbeforeunload
+   */
+  if (window.onbeforeunload !== undefined) {
+    return;
+  }
 
-    var prevented = false;
+  let prevented = false;
 
-    /*
-     * This catches the backspace key and asks the user whether they want to
-     * navigate back.
-     *
-     * Opera < 12.10 fires both keydown and keypress events, but keypress
-     * must return false. Opera >= 12.10 doesn't fire keypress for special
-     * keys, so keydown must return false. Regular event listeners and/or
-     * preventDefault don't work for this, they must be assigned directly
-     * to document.onkeydown and document.onkeypress.
-     */
-    document.onkeydown = function (event) {
-        if (event.keyCode == 8) {
-            var node = event.srcElement || event.target, tag = node.tagName.toLowerCase(),
-                type = (node.type || "").toLowerCase(),
-                prevent = !((tag == "input" && (type == "text" || type == "password")) || tag == "textarea");
+  /*
+   * This catches the backspace key and asks the user whether they want to
+   * navigate back.
+   *
+   * Opera < 12.10 fires both keydown and keypress events, but keypress
+   * must return false. Opera >= 12.10 doesn't fire keypress for special
+   * keys, so keydown must return false. Regular event listeners and/or
+   * preventDefault don't work for this, they must be assigned directly
+   * to document.onkeydown and document.onkeypress.
+   */
+  document.onkeydown = function (event) {
+    if (event.keyCode == 8) {
+      let node = event.srcElement || event.target; let tag = node.tagName.toLowerCase();
```

Comment:
Hi @monicaq21 , 

Just a small detail, i think `tag`'s declaration should be in a new line just like you did in [scripts/edit/ExampleRelationships.js](https://github.com/metabrainz/musicbrainz-server/blob/efc397bd5c4119f1c36dae86ab6f6281530c863e/root/static/scripts/edit/ExampleRelationships.js#L12#L17).

I'm not a maintainer so if i'm wrong just correct me.

Reply:
I think you can find all the similar problems by using **Ctrl + F** or **Command + F** then enter `; let` or `; const` in all the files you have changed. Of course if it is a problem.